build-xcframework.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env sh
+
+NAME="aries_askar"
+VERSION="0.2.8-dev.1"
```

Comment:
Can we pull the version number from elsewhere to avoid having another place to update it?